problem_builder/mentoring.py

```diff
@@ -1245,3 +1245,24 @@ def author_edit_view(self, context):
         fragment.add_javascript_url(self.runtime.local_resource_url(self, 'public/js/container_edit.js'))
         fragment.initialize_js('ProblemBuilderContainerEdit')
         return fragment
+
+    def student_view_data(self, context=None):
+        children_contents = []
+
+        for child_id in self.children:
+            child = self.runtime.get_block(child_id)
+
+            if child:
+                child_contents = child.student_view_data(context)
+                children_contents.append(child_contents)
+        return {
+            'self': {
```

Comment:
Makes sense. I did this to reflect the structure of the student view. I'll change it.